### DISCUSSION TITLE
How can I reduce carla resolution?

### DISCUSSION THREAD

**Discussion by 3F999** [carla-simulator/carla#6828] [Q&A] [answered]
CarlaUE4.sh -quality-level=Low 
doesnt work and gives me an error then just a black screen

**Comment by XiaoFei9704** [comment]
Don't U run like this? `./CarlaUE4.sh`

**Reply by 3F999** [reply]
I just can use this code : CarlaUE4.exe -dx11
otherweise it doesnt work at all

**Comment by 3F999** [comment, marked as answer]
Hi thank you from your answer, no actually I use carla version 9.13 and tryed to make my pathon script in a separate file like example files but after adding some vehicles and then wanted to destroy them by code but gave me some errors and after that Im not able even run predefined exampels like generate_traffic or manual_control. would you help me to solve this problem?
thanks in advance,
[Behnaz
![Screenshot 2023-10-16 143819](https://github.com/carla-simulator/carla/assets/134440584/0fa9c156-79ce-4609-8f82-03b522eed391)
